urbanbroccoli/laplacian.py

```diff
@@ -10,16 +10,33 @@ def create_laplacian_1d(nx, hx):
     return laplacian
 
 def create_laplacian_2d(nx, ny, hx, hy):
-    laplacian = zeros(shape=(nx, ny, nx, ny), dtype=float64)
+    '''Creates Laplacian matrix for 2-D Domain by discretizing the domain by grids of required sizes in X and Y direction.  
+       
+       It first creates a 4-dimensional array to show all components of laplace operator at each grid point.  
```

Comment:
The explanation is a bit verbose. And, please, do not use such long lines.

Reply:
Changes done.